docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guides/migration-guide-from-version-4-x-to-5-0/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "From version 4.x to 5.0"
+date: "2022-08-30"
+sidebar_position: -4
+---
+
+# Migration guide from version 4.x to 5.0
+
+Although the trackers underwent a huge internal rewrite (from Objective-C to Swift and from Java to Kotlin), the API was kept without breaking changes.
```

Comment:
Let's hope so! 😅
We did change some default configurations though. HTTPS rather than HTTP in Android. Think we standardised some others, I can have a look

Reply:
We have also standardised some configuration options. We discovered a couple of cases where the default configuration was defined differently in the `Emitter` class compared to the `EmitterConfiguration` or `NetworkConfiguration` classes. This means that trackers were not always configured as expected, depending on how they were created. The affected properties were `BufferOption` (`.DefaultGroup` in `Emitter`, `.Single` in `EmitterConfiguration`), `Protocol` (`.HTTP` in `Emitter`, `.HTTPS` in `NetworkConfiguration`) and `threadPoolSize` (2 in `Emitter`, 15 in `EmitterConfiguration`) on Android, and `BufferOption` (`SPBufferOptionDefaultGroup` in `Emitter`, `SPBufferOptionSingle` in `EmitterConfiguration`) on iOS.

The defaults are now consistently: 
`BufferOption` = DefaultGroup (maximum 10 events per batch)
`Protocol` = HTTPS
`threadPoolSize` = 15

Reply:
That's a good overview, but is it relevant for the users? The `Tracker` and `Emitter` classes were internal so they wouldn't know about the defaults there since they can't use them directly.

Reply:
Good point. It's only the BufferOption thing which might have affected people

Reply:
Added a note about the buffer option change!